app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> str:
+    cp_command, file_in_name, file_out_name = command.split()
+
+    if (
+            len(command.split()) < 3
```

Comment:
don't duplicate command "command.split()" twice
create a separated variable for it and call it on demand